README.md

```diff
@@ -28,20 +28,24 @@ See [installing.md][] for full installation instructions for Linux and Mac OS X.
 
 ## Prerequisites
 
-Before invoking gcsfuse, you must have a GCS bucket that you want to mount. If
+* Before invoking gcsfuse, you must have a GCS bucket that you want to mount. If
 your bucket doesn't yet exist, create one using the
 [Google Developers Console][console].
 
 [console]: https://console.developers.google.com
 
-GCS credentials are automatically loaded using [Google application default
+* Also, make sure [the Google Cloud Storage JSON API is enabled][enableAPI].
```

Comment:
Since this is now a list, it doesn't really make sense to have "Also, " here.

Reply:
Done.  thx for looking into this!